katas/content/distinguishing_unitaries/h_x/index.md

```diff
@@ -0,0 +1,8 @@
+**Input:** An operation that implements a single-qubit unitary transformation:
+either the Hadamard ($H$) gate
```

Comment:
The H gate in brackets looks a bit odd in the new rendering, let's just refer to it as H
```suggestion
either the $H$ gate
```